api/api-facade/api-http/src/main/java/org/eclipse/dirigible/api/v3/http/HttpClientFacade.java

```diff
@@ -195,6 +194,20 @@ private static void prepareHeaders(HttpClientRequestOptions httpClientRequestOpt
 		}
 	}
 
+	private static final String[] supportedTextMimeTypes = new String[] {
+			"application/CSV",
+			"application/csv",
+			"text/csv",
+			ContentType.TEXT_PLAIN.getMimeType(),
+			ContentType.TEXT_HTML.getMimeType(),
+			ContentType.TEXT_XML.getMimeType(),
+			ContentType.APPLICATION_JSON.getMimeType(),
+			ContentType.APPLICATION_ATOM_XML.getMimeType(),
+			ContentType.APPLICATION_XML.getMimeType(),
+			ContentType.APPLICATION_XHTML_XML.getMimeType()
+	};
+	private static final Set<String> recognizedTextMimeTypes = new HashSet<>(Arrays.asList(supportedTextMimeTypes));
```

Comment:
It's not really needed to be `Set<String>`. `List<String>` would be also fine, as it is `private static final` and the values are set by us, so I guess no duplication would be possible.

Reply:
It's not about the duplication, it would be a bit more performant to have it as a HashSet as it is used for checking if a content type is supported by us